lib/generators/clearance/install/install_generator.rb

```diff
@@ -20,7 +20,7 @@ def inject_clearance_into_application_controller
       end
 
       def create_or_inject_clearance_into_user_model
-        if File.exists? 'app/models/user.rb'
+        if File.exist? 'app/models/user.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.